### PR TITLE
Bioprinter changes

### PR DESCRIPTION
**Pull request by Trilbyspaceclone**
Inconstantly upto half reduced all church prints 
changes biomatters 50% reduction in printing to a reduction of 25%

Removes (rather then improves) church manifests for medical items and weapons do to exploits

### CODE REVIEW

File: code/modules/core_implant/cruciform/rituals/construction.dm [hunk 1]
```diff
@@ -607,7 +607,7 @@ GLOBAL_LIST_INIT(nt_constructs, init_nt_constructs())
 	name = "Absolutism Medkit"
```
Comment (SingingSpock) [inline code/modules/core_implant/cruciform/rituals/construction.dm:607]: Could you actually comment out this recipe? It's a good idea, but I haven't figured out how to stop our resident powergamers from printing it with the printer and then Uprooting it to get free mats

**Review by SingingSpock — CHANGES_REQUESTED** [review]

File: code/modules/clothing/suits/armor.dm [hunk 2]
```diff
@@ -412,7 +412,7 @@
 	body_parts_covered = UPPER_TORSO|LOWER_TORSO|LEGS|ARMS
 	armor_list = list(melee = 40, bullet = 40, energy = 40, bomb = 50, bio = 100, rad = 100)
 	flags_inv = HIDEJUMPSUIT
-	matter = list(MATERIAL_PLASTEEL = 60, MATERIAL_PLASTIC = 8, MATERIAL_SILVER = 5, MATERIAL_GOLD = 5)
+	matter = list(MATERIAL_PLASTEEL = 30, MATERIAL_PLASTIC = 5, MATERIAL_SILVER = 5, MATERIAL_GOLD = 5)
```
[the remark this answers is not in the document]
Comment (Trilbyspaceclone) [reply code/modules/clothing/suits/armor.dm:415]: its the same amount of plasteel as its always been to print for the church biolathen

File: code/datums/autolathe/biomatter.dm [hunk 3]
```diff
@@ -193,7 +193,7 @@
 
 /datum/design/bioprinter/nt_clothes/knight_rig
 	name = "Knight Hardsuit Module"
-	materials = list(MATERIAL_BIOMATTER = 100, MATERIAL_GOLD = 10, MATERIAL_SILVER = 10, MATERIAL_PLASTEEL = 100)
+	materials = list(MATERIAL_BIOMATTER = 70, MATERIAL_GOLD = 5, MATERIAL_SILVER = 5, MATERIAL_PLASTEEL = 50)
```
[the remark this answers is not in the document]
Comment (Trilbyspaceclone) [reply code/datums/autolathe/biomatter.dm:196]: its 50% + 25%
your accually getting a better deal on this print then before

[the remark this answers is not in the document]
Comment (SingingSpock) [reply code/datums/autolathe/biomatter.dm:196]: Yes, that's the issue. New price is too low imo.

File: code/modules/clothing/suits/armor.dm (hunk 2 above)
[the remark this answers is not in the document]
Comment (SingingSpock) [reply code/modules/clothing/suits/armor.dm:415]: Previous cost with a bioprinter: 50% of 60 plasteel: 30 plasteel. New cost with a bioprinter: 75% of 30 plasteel: 22.5 plasteel. I recommend a new price of 40, as that will keep the cost with bioprinter unchanged

Comment (Trilbyspaceclone) [inline code/modules/clothing/suits/armor.dm:415]:
```suggestion
	matter = list(MATERIAL_PLASTEEL = 40, MATERIAL_PLASTIC = 5, MATERIAL_SILVER = 5, MATERIAL_GOLD = 5)
```

File: code/modules/clothing/suits/armor.dm [hunk 4]
```diff
@@ -478,7 +478,7 @@
...
```
Comment (Trilbyspaceclone) [inline code/modules/clothing/suits/armor.dm:481]:
```suggestion
	matter = list(MATERIAL_PLASTEEL = 40, MATERIAL_PLASTIC = 5, MATERIAL_SILVER = 5, MATERIAL_GOLD = 5)
```

File: code/modules/projectiles/guns/energy/laser/laser.dm [hunk 5]
```diff
@@ -10,7 +10,7 @@
 	w_class = ITEM_SIZE_BULKY
 	force = WEAPON_FORCE_NORMAL
 	origin_tech = list(TECH_COMBAT = 3, TECH_MAGNET = 2)
-	matter = list(MATERIAL_PLASTEEL = 20, MATERIAL_WOOD = 8, MATERIAL_SILVER = 10)
+	matter = list(MATERIAL_PLASTEEL = 15, MATERIAL_WOOD = 8, MATERIAL_SILVER = 1)
```
Comment (Trilbyspaceclone) [inline code/modules/projectiles/guns/energy/laser/laser.dm:13]:
```suggestion
	matter = list(MATERIAL_PLASTEEL = 15, MATERIAL_WOOD = 8, MATERIAL_SILVER = 6)
```

File: code/datums/autolathe/biomatter.dm (hunk 3 above)
Comment (Trilbyspaceclone) [inline code/datums/autolathe/biomatter.dm:196]:
```suggestion
	materials = list(MATERIAL_BIOMATTER = 70, MATERIAL_GOLD = 7, MATERIAL_SILVER = 7, MATERIAL_PLASTEEL = 70)
```

**Pull request merged**